target/linux/mvebu/patches-4.19/302-Armada-8040-clearfog-gt-8k.dts.patch

```diff
@@ -0,0 +1,475 @@
+diff --git a/arch/arm64/boot/dts/marvell/Makefile b/arch/arm64/boot/dts/marvell/Makefile
```

Comment:
This should be a series of backported patches with proper messages from Linus tree or dts file dropped in `target/linux/mvebu/files-4.19/arch/arm64/boot/dts/marvell`.